xchange-okcoin/src/main/java/info/bitrich/xchangestream/okcoin/OkCoinStreamingExchange.java

```diff
@@ -60,3 +60,4 @@ public StreamingMarketDataService getStreamingMarketDataService() {
     @Override
     public void useCompressedMessages(boolean compressedMessages) { streamingService.useCompressedMessages(compressedMessages); }
 }
+
```

Comment:
nit: try to avoid these sorts of whitespace/formatting changes in PRs. It makes the scope of a change harder to assess.